packages/datagateway-common/src/detailsPanels/dls/visitDetailsPanel.component.tsx

```diff
@@ -47,6 +47,7 @@ const VisitDetailsPanel = (
   const { rowData, detailsPanelResize } = props;
   const [t] = useTranslation();
   const { data } = useInvestigationDetails(rowData.id);
+  //TODO: Migrate to using entity size prop once dls has migrated to ICAT v5
```

Comment:
This is the only place useInvestigationSize() query is used, can be removed once dls has migrated to ICAT V5